package.json

```diff
@@ -6,7 +6,7 @@
   "main": "app.js",
   "scripts": {
     "start": "node app.js",
-    "dev-start": "source ./env/development && source ./env/dev-secrets && node --debug app.js",
+    "dev-start": "source ./env/development && source ./env/dev-secrets && node --inspect app.js",
```

Comment:
good idea.  yeah, would love for this to just work so we can scrap node-inspector which has been very buggy for me in the past. (it works now though so gotta make due)